simphony_metadata.yml

```diff
@@ -259,15 +259,14 @@ CUDS_KEYS:
     CUBA.SURFACE_TENSION:
       default: 0.07
       scope: CUBA.USER
-      shape: (2)
     CUBA.MATERIAL:
       shape: (2)
 
   DIRICHLET_BOUNDARY_CONDITION:
     definition: Dirichlet boundary condition
     parent: CUBA.CONDITION
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLES:
+    variables:
```

Comment:
Same as my previous comment. For plurals, we have to define something like CUBA.VARIABLE and describe it with extra metadata attributes such as shape. Anything with no `CUBA.` prefix implies a fixed metadata-level attribute. However, I think we should work on this whole thing and improve it.